src/parity_testing/auth/testSuites/signOut/Test_that_global_signOut_error_returns_partial_success_with_global_sign_out_error.json

```diff
@@ -27,12 +27,13 @@
         {
             "type": "amplify",
             "apiName": "signOut",
-            "responseType": "complete",
+            "responseType": "success",
             "response": {
+                "signOutResult": "PARTIAL",
                 "globalSignOutError": {
                     "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyLCJvcmlnaW5fanRpIjoib3JpZ2luX2p0aSJ9.Xqa-vjJe5wwwsqeRAdHf8kTBn_rYSkDn2lB7xj9Z1xU",
                     "exception": {
-                        "errorType": "GlobalSignOutException",
+                        "errorType": "NotAuthorizedException",
```

Comment:
iOS doesn't have `GlobalSignOutException`, so can we use `NotAuthorizedException` instead